docs/conf.py

```diff
@@ -37,11 +37,21 @@
 # needs_sphinx = '1.2'
 
 intersphinx_mapping["pypa-packaging"] = ("https://packaging.python.org/en/latest/", None)  # noqa
-intersphinx_mapping["asdf"] = ("https://asdf.readthedocs.io/en/latest/", None)  # noqa
-intersphinx_mapping["asdf-standard"] = ("https://asdf-standard.readthedocs.io/en/latest/", None)  # noqa
 intersphinx_mapping["asdf-astropy"] = ("https://asdf-astropy.readthedocs.io/en/latest/", None)  # noqa
 intersphinx_mapping["pytest"] = ("https://docs.pytest.org/en/latest/", None)  # noqa
 
+subprojects = {
```

Comment:
Is this variable special for sphinx/rtd? If so can you point me to docs describing what it needs to be? I'm curious to know if this needs to be updated to by in-sync across all subprojects.

Reply:
Thanks!